queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Great work setting up the animal table, you have done really well, I have just one suggestion the query `WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';`  only selects animals with date of birth before January 1, 2019, but excludes those born on the same day as January 1, 2019.

To include all animals born in 2019, you need to change the upper limit of the date range to '2019-12-31' :